test/uk/gov/hmrc/tai/service/IncomeServiceSpec.scala

```diff
@@ -226,6 +230,53 @@ class IncomeServiceSpec extends BaseSpec {
       val sut = createSUT(employmentService = mockEmploymentService, incomeRepository = mockIncomeRepository)
       val result = sut.taxCodeIncomes(nino, TaxYear())(HeaderCarrier()).futureValue
 
+      result mustBe taxCodeIncomes
+    }
+
+    " not use the employments API if toggle is disabled" in {
```

Comment:
is there tests for the two possible cases of Seq.empty[Employment]? And im guessing there is tests to verify the use of employments API?

Reply:
It should be covered by this one: 
https://github.com/hmrc/tai/blob/27cb651ae98c36480fe007400f1fcbc9654c36b9/test/uk/gov/hmrc/tai/service/IncomeServiceSpec.scala#L227

Reply:
nice one